Source/ablastr/fields/PoissonSolver.H

```diff
@@ -153,6 +159,27 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
                 {{ beta[0], beta[1], beta[2] }};
 #endif
 
+#if (defined(WARPX_USE_PSATD)  && defined(WARPX_DIM_3D))
+        // Use the Integrated Green Function solver (FFT) on the coarsest level if it was selected
```

Comment:
Small update I would suggest:

- If FFTs were not compiled in, but `is_solver_multigrid=false` is passed to this function, then this falls back to MLGM solve for all levels without raising a warning.
- I realized the `is_solver_multigrid` name is a but confusing, because we do the IGF only on level 0 and an MLMG on all others in MR. Maybe we find a good alternative name.

Reply:
The first point does not really apply to WarpX, I believe. 
There's an assert that checks if FFTs are compiled when the IGF solver is selected. 
It's [here](https://github.com/ECP-WarpX/WarpX/pull/4648/files#diff-1450fa5afa66d2e4dde0bd48fe21eb084d339554ad937d3604376713e053e3c4R774-R778). 
Anyway, I added a similar assert in ablastr in the follow-up PR #4945. 
Let me know if this solution is ok for you.

Reply:
T### hanks! Yes, we need the check in ABLASTR directly because that is a library that we also call from ImpactX :pray: